spec/support/controller_exceptions.rb

```diff
@@ -0,0 +1,23 @@
+module ControllerExceptions; end
+
+# Test group helpers for allowing exceptions from controller tests to be caught by specs.
+module ControllerExceptions::TestGroupHelpers
+  BYPASS_RESCUE_CONSTANT_NAME = :__controller_exceptions_test_group_helpers_bypass_rescue
+
+  def self.extended(module_)
+    module_.module_eval do
+      let(BYPASS_RESCUE_CONSTANT_NAME) { true }
+      before(:each) do
+        bypass_rescue if send(BYPASS_RESCUE_CONSTANT_NAME)
```

Comment:
that's the default. I find that I have a lot of controller tests that do `expect { action }.to raise_exception()`

anyway I think this is a better default. it doesn't make sense to test for HTTP response codes from the controller.